### PR TITLE
Task tiny JS

### PR DESCRIPTION
**Pull request by SamVal007**
A tiny js world
Please, review

[demo](https://samval007.github.io/a-tiny-JS-world/)
[codebase](https://github.com/SamVal007/a-tiny-JS-world)

### CODE REVIEW

**Comment by OleksiyRudenko** [comment]
This PR contains commits from other tasks.
You might have missed warning under section B1 heading and also ignored step B122 in the submission instructions.
Please fix.

**Comment by SamVal007** [comment]
> This PR contains commits from other tasks.
> You might have missed warning under section B1 heading and also ignored step B122 in the submission instructions.
> Please fix.

Thank you!
Done, I hope!

File: submissions/SamVal007/a-tiny-JS-world/index.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+
+
+// ======== OUTPUT ========
+
+[man, dog, woman, cat, catWoman].forEach(elem => {
+   print(`Hi! I am ${elem.species}; My name is ${elem.name};
+     My gender is ${elem.gender}; I have ${elem.legs} legs;
+     I have ${elem.hands} hands; We are friends with ${elem['friends'].map(friend => friend.name).join(', ')} and I want to say ${elem.speach} to them`);
+});
```
Comment (OleksiyRudenko) [inline submissions/SamVal007/a-tiny-JS-world/index.js:66]: Topics for you to reflect on:
1. What if an inhabitant has got no friends?
2. Now output is multiline for the sake of code readability. What would you change if we still want the code well fomatted yet output wouldn't be multiline? Think of a couple of options. One extra option might be an array of the resulting string fragments (logical components) joined then together.

**Review by OleksiyRudenko — APPROVED** [review]
@SamVal007 great job!
Check the comment below for the ideas to think about.

**Pull request merged**